packages/stockflux-launcher/src/reducers/free-text-search/FreeTextSearch.test.js

```diff
@@ -0,0 +1,58 @@
+import reducer, { initialSearchState } from './FreeTextSearch';
+import action from './Action';
+
+describe('Action Reducer', () => {
+  it('Should return inital state when given an unknown action', () => {
```

Comment:
I'd imagine this should throw, unknown actions are a no no, especially if you're using constants for the action names.

Reply:
- Changed reducers to throw errors on any unknown action. 
- Tests updates to check for thrown errors.